src/components/Customers/TradeLinkCell.svelte

```
@@ -0,0 +1,68 @@
+<script lang="ts">
+  import { token } from '@/lib/s2sStore';
+  import type SystemCustomer from '@/models/customer/customer';
+  import ChannelsRepo from '@/repos/channels-repo';
+  import Icon from '../Icons/Icon.svelte';
+  import { useQuery } from '@sveltestack/svelte-query';
+  import IconExternalLink from '../Icons/IconExternalLink.svelte';
+  import { getContext, onMount } from 'svelte';
+  import type CustomerToken from '@/models/customer/customer-token';
+
+  export let data: SystemCustomer;
+  let loading = false;
+  const dependencyContainer: any = getContext('trade-link-cell-dependencies');
+
+  onMount(() => {
+    if (!dependencyContainer) console.error('Ensure trade-link-cell-dependencies context is set.');
```

And again - I'd rather strip this check out and let it fail